### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by faddat**
# PR Details

Update:

Branding, community, compile instructions, ask users to file issues or make forum posts, describe project in one sentence.

## Description

This PR updates readme.md to match new documentation in the Wiki. 

## Types of changes

-   [x]   Docs change / refactoring / dependency upgrade
-   [ ]   Bug fix (non-breaking change which fixes an issue)
-   [ ]   New feature (non-breaking change which adds functionality)
-   [ ]   Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

-   [ ]   My code follows the code style of this project.
-   [ ]   My change requires a change to the documentation.
-   [x]   I have updated the documentation accordingly.
-   [ ]   I have added tests to cover my changes.
-   [ ]   All new and existing tests passed.

### CODE REVIEW

**Comment by sunnyking** [comment]
Project name is 'V Systems', not 'v.systems'. v.systems is only our website domain name.

**Comment by faddat** [comment]
@sunnyking I've updated the project name to "V Systems"

Cheers!

File: README.md [hunk 1]
```diff
@@ -1,55 +1,30 @@
...
-```
-sbt -Dnetwork=testnet packageAll
-```
+We maintain an extensive [Wiki](https://github.com/virtualeconomy/v-systems/wiki) that covers most topics.  If there's something you need to know that isn't covered there, please ask us either by filing an issue [here](https://github.com/virtualeconomy/v-systems/issues) or make a post on our [forum](https://forum.v.systems/).  We're happy to help with anything!
 
-## Running Tests
+## Compiling Packages from source
 
-Execute 
+We've written a [guide](https://github.com/virtualeconomy/v-systems/wiki/Compile-v.systems-from-source-code) that covers compiling and installing v.systems on Ubuntu 18.04.
```
Comment (sunnyking) [inline README.md:21]: one more here

**Comment by faddat** [comment]
Should now be resolved.

**Comment by ncying** [comment]
@faddat still has some usage of v.systems in wiki

**Comment by faddat** [comment]
@ncying thanks, I'll eliminate those, as well.

**Pull request merged**